pom.xml

```diff
@@ -32,7 +32,7 @@
 		<quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
 		<quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
 		<quarkus.platform.version>${quarkus.version}</quarkus.platform.version>
-		<curation.version>v0.34.2</curation.version>
+		<curation.version>v0.35.0-rc2</curation.version>
```

Comment:
This is not production code... it needs to be `v0.35.0`

Reply:
Version v0.35.0 does not exist in maven central yet.